kiwi/tasks/image_info.py

```diff
@@ -101,7 +101,17 @@ def process(self):
             solver = self._setup_solver()
             package_list = self.xml_state.get_bootstrap_packages() + \
                 self.xml_state.get_system_packages()
-            solved_packages = solver.solve(package_list)
+            bootstrap_packages = solver.solve(
+                self.xml_state.get_bootstrap_packages(), False,
+                True if self.xml_state.get_bootstrap_collection_type() is
+                'onlyRequired' else False
+            )
+            solved_packages = solver.solve(
+                self.xml_state.get_system_packages(), False,
+                True if self.xml_state.get_system_collection_type() is
+                'onlyRequired' else False
+            )
+            solved_packages.update(bootstrap_packages)
```

Comment:
Yes this is right, I made the solving in two operations to be able to apply different criteria for bootstrap and image packages.

I did some tests to validate the result was the same, but I'll make some more accurate tests, as I haven't done it with a complete image including the recommended packages everywhere.